src/amo/css/AddonReviewList.scss

```diff
@@ -10,7 +10,13 @@
 
 .AddonReviewList-header {
   margin: 15px 0;
+  display: -ms-flexbox;
+  display: -webkit-flex;
   display: flex;
+  -ms-flex-align: center;
```

Comment:
Thanks! And what do you think about the [image above](https://github.com/mozilla/addons-frontend/pull/2609#issuecomment-310301144)? That's the alignment we need for the icons right?